app/models/favorite_handler.rb

```diff
@@ -0,0 +1,13 @@
+class FavoriteHandler < ActiveRecord::Base
```

Comment:
I wanted to be able to say ```user.favorites``` to get a list of a user's favorite posts. In setting up the many to many association, I didn't think that my method could match the controller name because I'd have a line that says ```has_many :favorites, though: :favorites```. There are probably better names to use here, but that's why I decided on that name.